src/main/java/org/minions/devfund/carlosorellana/Squarelotron.java

```diff
@@ -0,0 +1,204 @@
+package org.minions.devfund.carlosorellana;
+
+/**
+ * The goal is execute the Squarelotron methods.
+ *
+ * @author Carlos Orellana.
+ * @since 07/17/2018.
+ */
+public class Squarelotron {
+
+    private int[][] squarelotron;
+    private int size;
+
+    /**
+     * Gets Squarelotron Matrix.
+     *
+     * @return Squarelotron Matrix.
+     */
+    public int[][] getSquarelotron() {
+        return squarelotron.clone();
+    }
+
+    /**
+     * Sets Squarelotron Matrix.
+     *
+     * @param squarelotron - Squarelotron Matrix.
+     */
+    public void setSquarelotron(final int[][] squarelotron) {
+        this.squarelotron = squarelotron;
+    }
+
+    /**
+     * Constructor Sqarelotron.
+     *
+     * @param n - Size of squarelotron array.
+     */
+    public Squarelotron(final int n) {
+        this.size = n;
+        this.squarelotron = new int[size][size];
+
+        for (int i = 0; i < size; i++) {
+            for (int j = 0; j < size; j++) {
+                squarelotron[i][j] = size * i + j + 1;
+            }
+        }
+    }
+
+    /**
+     * Gets the maximum number of ring in the squarelotron.
+     *
+     * @return Numbers maximum of rings.
+     */
+    public int getMaxRings() {
+        return (this.size % 2 == 0) ? size / 2 : size / 2 + 1;
+    }
+    /**
+      * Modifies A Upside-Down Flip of the outer ring of Squarelotron.
+      *
+      * @param ring - Number of ring.
+      * @return Instance new Squarelotron.
+      */
+    public Squarelotron upsideDownFlip(int ring) {
+
+        if (ring > this.getMaxRings() || ring < 1) {
+            throw new NumberFormatException();
+        }
+
+        Squarelotron squarelotron = new Squarelotron(this.size);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "squarelotron" which hides the field declared at line 11. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)